datamodels/attestation-results/grammar/examples/teep-json-1.diag

```diff
@@ -0,0 +1,19 @@
+{
+  "eat_profile": "tag:github.com/veraison/ar4si,2022-10-17",
+  "ear.status": "affirming",
+  "ear.trustworthiness-vector": {
+    "instance-identity": 2,
+    "configuration": 2,
+    "executables": 2,
+    "hardware": 2
+  },
+  "iat": 1666529284,
+  "ear.appraisal-policy-id": "https://veraison.example/policy/1/60a0068d",
```

Comment:
Just a note: at the moment, we do not expose policies externally via a URL. The policy ID utilised internally and reported in the result is a URI, with the scheme defined by the policy agent used (and therefore the expected rules format), and the tenant ID as the authority. So, at the moment, this would look more like `"opa://7"`. 

Once we define https://github.com/veraison/services/issues/16, that might change (e.g. we may fine an API endpoint for accessing policies, or require that an external URL is supplied on registration)...